### PR TITLE
 [V4] Unify Appender.Layout.format delegate type with the rest of Ocean

### PR DESCRIPTION
**Pull request by mathias-lang-sociomantic**
The first commit cleans up the indent / code style in the modules being touched. Since I was changing some part of the code, it felt counter-productive to spend time keeping the original, out of sync style. As those modules are seldom touched, so it's pretty safe conflict-wise.

The second commit is just a clean-up I spotted when doing the original logger separation / refactoring.

### CODE REVIEW

**Comment by mathias-lang-sociomantic** [comment]
CC @daniel-zullo-sociomantic : Your app might be the only one affected by this AFAIK.
Could you check if this change would be beneficial to you ?

**Comment by mathias-lang-sociomantic** [comment]
Ping

**Review by mihails-strasuns-sociomantic — APPROVED** [review]
Note to self: will require another turtle update

**Comment by daniel-zullo** [comment]
@mathias-lang-sociomantic Sorry, I couldn't check it yet I'll do it ASAP (the changes LGTM though). The PR was already merged and I'll report back if I find any issue.

By the way, it seems one of the commits didn't passed the tests. Maybe it's related to the turtle update?

**Comment by mihails-strasuns-sociomantic** [comment]
> By the way, it seems one of the commits didn't passed the tests

It did, the red cross was because PR was affecting code with low code coverage

**Comment by daniel-zullo** [comment]
I've tested this patch in my app and everything went well. I tested this patch independently cherry-picking the commits to ocean `v3.7.1` as I couldn't advance the app to ocean`v4.x.x` because it involves many other changes in the app.

File: relnotes/layout-format.migration.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## Appender.Layout's `format` method sink type was updated
+
+* `ocean.util.log.Appender : Appender.Layout.format`
```
Comment (leandro-lucarella-sociomantic) [inline relnotes/layout-format.migration.md:3]: This formatting is not correct :grumpy: (it should be without the `*`)

**Pull request merged**